otter/models/cass.py

```diff
@@ -443,11 +443,27 @@ def _jsonloads_data(raw_data):
     return data
 
 
+def _group_status(status, deleting):
+    if deleting:
+        return ScalingGroupStatus.DELETING
+    else:
+        # TODO: #1304
+        if status is None:
+            return ScalingGroupStatus.ACTIVE
+        elif status == 'DISABLED':
+            return ScalingGroupStatus.ERROR
+        else:
+            return ScalingGroupStatus.lookupByName(status)
+
+
 def _unmarshal_state(state_dict):
     desired_capacity = state_dict['desired']
     if desired_capacity is None:
         desired_capacity = 0
 
+    status = _group_status(state_dict['status'],
+                           state_dict.get('deleting', False))
```

Comment:
So, this might be a bit weird. Since some queries don't fetch 'deleting' it won't always be in this state_dict, so in those cases I'm assuming we've got an ACTIVE group. any alternative suggestions?